gpkit/nomials/variables.py

```diff
@@ -148,6 +148,8 @@ def __new__(cls, shape, *args, **descr):
             i = it.multi_index
             it.iternext()
             descr.update({"idx": i})
+            assert value_option is not None
+            # TODO refactor logic for value_option and remove above assert
```

Comment:
@galbramc and @whoburg , by the construction of the code, `value_option` is guaranteed to have a value which makes sense if `values` is truthy.